### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_SelectSource-Target_SchemaCompare_AI4D: Name property must not contain special characters as '...' for more commands button.

### ISSUE THREAD

**Issue by ShwetaBaglane** [microsoft/azuredatastudio#6730]
[“Check out Accessibility Insights! !](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yMAS; #A11yTCS; #A11y_ADS_Schema_Dacpac_Backup; #Azure Data Studio; #DesktopApp; #Win10; #AI4D; #MAS4.1.2; 

**Environment Details:**
Application Name: Azure Data Studio
Application Version: 1.10.0-insider (user setup)
Commit: 975e45061b073ecce2398d00c12f525d90e07b02
Date: 2019-08-08T19:22:13.572Z
VS Code 1.37.0
Electron: 4.2.7
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
Nvda Version: 2019.2rc2

**Additional Details:**
Tools Name: Nvda
MAS References: MAS4.1.2

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Schema Compare extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Schema compare from Context menu.
8. Launch Accessibility Insights tool and hover the mouse over More commands button represented as '...'

**Actual:**
**Issue 1:** Name property must not contain special characters as '...' for 'Select Source' and 'Select Target' buttons.
**Issue 2:** Tooltip not displayed on keyboard navigation to 'Select Source' and 'Select Target' buttons.

**Expected:**
**Issue 1:** Name for 'Select Source' and 'Select Target' buttons should be available.
**Issue 2:** Tooltip should be displayed on keyboard navigation to 'Select Source' and 'Select Target' buttons.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: (https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
Screen reader users will not get to know the context of the button if it got announced as special characters.

**MAS Reference:**
MAS 4.1.2-
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Attachment for Reference:**
![image](https://user-images.githubusercontent.com/53556874/62942224-78a18800-bdf5-11e9-88d7-7bc4d0257f0b.png)

![image](https://user-images.githubusercontent.com/53556874/62942148-5576d880-bdf5-11e9-8d96-6852ff6af743.png)

**Comment by fsteffi** [comment]
Issue1 has been solved but issu2:Tooltip not displayed on keyboard navigation to 'Select Source' and 'Select Target' buttons is still present in the application.
[#6730A11y_AzureDataStudio(Schema,Dacpac,Backup)_SelectSource-Target_SchemaCompare_AI4D.pptx](https://github.com/microsoft/azuredatastudio/files/3712081/6730A11y_AzureDataStudio.Schema.Dacpac.Backup._SelectSource-Target_SchemaCompare_AI4D.pptx)

**Comment by Benjin** [comment]
@fsteffi I haven't been able to find any location where tooltips appear when using keyboard navigation.  I've checked other places in this extension, the Dacpac extension, or ADS stock functionality.  Are you sure that's expected/normal behavior?

**Comment by udeeshagautam** [comment]
Just additional data point : Even in VSCode mouse navigation shows tool tips but tabs don't show tool tips.  

on mouse (tool tip)

![image](https://user-images.githubusercontent.com/46980425/66606934-fa98fd80-eb67-11e9-80d9-260c8de85460.png)

on keyboard tab (no tool tip)
![image](https://user-images.githubusercontent.com/46980425/66606974-10a6be00-eb68-11e9-884d-559898e5a46b.png)

**Comment by fsteffi** [comment]
@Benjin yes the expected behaviour is that the tooltip is visible even while navigating to the button using keyboard, such that the keyboard only users receive the information related to the button that is available while mouse hover .

**Comment by Benjin** [comment]
@fsteffi Thanks for the confirmation.  I'm unable to find any other location where this is the case.  Can you record a video demonstrating somewhere in ADS where this occurs so I have a starting point for my investigation?

**Comment by udeeshagautam** [comment]
Since the control is common enough and not destructive, we can mitigate this with proper documentation. Also we should open the corresponding behavior bug on VSCode.

**Comment by udeeshagautam** [comment]
Changed the assignment before documentation by mistake.

**Comment by udeeshagautam** [comment]
Added external VSCode bug : https://github.com/microsoft/vscode/issues/83245

**Comment by kenvanhyning** [comment]
Closing as upstream
